docs/applications/simple-bank-account/contract/src/main/java/com/scalar/application/bankaccount/contract/AccountHistory.java

```diff
@@ -41,8 +41,8 @@ public JsonNode invoke(Ledger<JsonNode> ledger, JsonNode argument, @Nullable Jso
               getObjectMapper()
                   .createObjectNode()
                   .put("id", asset.id())
-                  .put("data", asset.data().asText())
-                  .put("age", asset.age());
+                  .put("age", asset.age())
+                  .set("data", asset.data());
```

Comment:
FYI (For reviewers):

In this sample application, the `balance` data is stored as an asset data as follows.

```sql
postgres=# select id, age, output from scalar.asset;
  id  | age |     output
------+-----+-----------------
 a111 |   0 | {"balance":0}
 b222 |   0 | {"balance":0}
 a111 |   1 | {"balance":100}
 a111 |   2 | {"balance":0}
 b222 |   1 | {"balance":100}
(5 rows)
```

And, `asset.data()` returns `com.fasterxml.jackson.databind.JsonNode` when we use Jackson.
https://javadoc.io/doc/com.scalar-labs/scalardl-java-client-sdk/latest/index.html

In this case, it seems that we should use `set(String propertyName, JsonNode value)` to set `JsonNode` as value instead of using `put(String fieldName, String v)`. 

- put(String fieldName, String v)
  - https://javadoc.io/static/com.fasterxml.jackson.core/jackson-databind/2.17.0/com/fasterxml/jackson/databind/node/ObjectNode.html#put-java.lang.String-java.lang.String-
- set(String propertyName, JsonNode value)
  - https://javadoc.io/static/com.fasterxml.jackson.core/jackson-databind/2.17.0/com/fasterxml/jackson/databind/node/ObjectNode.html#set-java.lang.String-com.fasterxml.jackson.databind.JsonNode-

(I'm not familiar with `Jackson`, but according to the test result, it seems that the `put(String fieldName, String v)` method does not show expected result even if we set `JsonNode` as String data by using `.asText()` method.)

Therefore, to set `JsonNode` in the contract result (i.e., to set `balance` data in `ArrayNode result`), it seems that we have to use `set(String propertyName, JsonNode value)`.

Reply:
> (I'm not familiar with Jackson, but according to the test result, it seems that the put(String fieldName, String v) method does not show expected result even if we set JsonNode as String data by using .asText() method.)

@kota2and3kan 
Thank you for testing this. I think it's expected because `asText()`

> will return a valid String representation of the container value, **if the node is a value node** (method [JsonNode.isValueNode()](https://javadoc.io/static/com.fasterxml.jackson.core/jackson-databind/2.17.0/com/fasterxml/jackson/databind/JsonNode.html#isValueNode--) returns true), **otherwise empty String**.

https://javadoc.io/static/com.fasterxml.jackson.core/jackson-databind/2.17.0/com/fasterxml/jackson/databind/node/ContainerNode.html#asText--

So, we should not use it to stringify an `ObjectNode` or `ArrayNode`.

Reply:
@jnmt 
Ah, I see. I didn't look into `asText()`. Thank you for the information! 🙇‍♂️